etc/import-app

```diff
@@ -185,26 +205,31 @@ elif [[ "$import" =~ ^[0-9]+$ ]] || [[ "$import" =~ *'://'*'/pull/'[0-9]+$ ]];th
   for app in $imported_apps ;do
     cp -af "$(pwd)/pi-apps/apps/$app" "${DIRECTORY}/apps"
     echo "Copied '$(pwd)/pi-apps/apps/${app}' to ${DIRECTORY}/apps"
+   
+    #add to Imported category
+    if ! grep -q "$app" "${DIRECTORY}/etc/categories" && ! grep -q "$app" "${DIRECTORY}/data/category-overrides"; then
+	  echo "$app|Imported" >> "${DIRECTORY}/data/category-overrides"
+    fi
   done
   
 else
   #unrecognized input
   error "Unrecognized input '$import'"
 fi
 
-echo -e "\n$(basename "$0") just imported these apps:"
-echo "$imported_apps" | sed 's/^/- /g'
 
 #display on-screen which apps were updated
 IFS=$'\n'
 for app in $imported_apps ;do
   LIST="$LIST
-$([ -f "${DIRECTORY}/apps/$app/icon-64.png" ] && echo "${DIRECTORY}/apps/$app/icon-64.png" || echo "${DIRECTORY}/icons/none-64.png")
+$([ -f "${DIRECTORY}/apps/$app/icon-24.png" ] && echo "${DIRECTORY}/apps/$app/icon-24.png" || echo "${DIRECTORY}/icons/none.png")
```

Comment:
Is there a reason we are displaying imported apps with 24x24 icons rather than 64x64 icons?

Reply:
I think 64x64 is too big for the app name text size beside it, so I think 24x24 will look nicer.

Reply:
Okay.